modules/custom/az_core/css/az-core-chosen.css

```diff
@@ -0,0 +1,5 @@
+.chosen-container,
+.chosen-search-input,
```

Comment:
<img width="1590" alt="image" src="https://github.com/user-attachments/assets/96a30d4d-1ccb-4b16-9bde-af9256ba2c81" />
<p> </p>

To fix the issue @trackleft identified, maybe we can skip applying this style to `.chosen-search-input`?

```suggestion
```